corehq/apps/app_manager/signals.py

```diff
@@ -34,9 +36,17 @@ def update_callcenter_config(sender, application, **kwargs):
         notify_exception(None, "Error updating CallCenter config for app build")
 
 
+def expire_latest_enabled_build_profiles(sender, application, **kwargs):
```

Comment:
Nit: This happens whenever the app is saved, but the cache only needs to be expired when a new build is created (or would this feature apply to live preview or that thing where you load the latest saved version of the application?)

Reply:
i have added the check `application.copy_of` which would run only if this is a build doc and is saved which is going to be a rare occasion.